server.py

```diff
@@ -381,6 +382,112 @@ def tilejson(fmt, tile_pixel_size=None):
     return resp
 
 
+def t2_meta_and_offset(requested_tile, materialized_zooms, metatile_size):
+    # Find the materialized zoom that holds this tile
+    mz = next(x for x in sorted(materialized_zooms, reverse=True) if x <= requested_tile.z)
```

Comment:
This looks like it will just throw `StopIteration` if `requested_tile.z` is out of range. I don't think this will happen in regular use, but might make it confusing to track down if we misconfigure it (probably locally). Do you think it could be wrapped in a `try: except:` to add information to the exception?

Reply:
Yes, good catch!